source/Octopus.Tentacle/Kubernetes/PodLogLineParser.cs

```diff
@@ -62,23 +62,42 @@ static class PodLogLineParser
 
         public static PodLogLineParseResult ParseLine(string line, IPodLogEncryptionProvider encryptionProvider)
         {
-            var logParts = line.Split(new[] { '|' }, 4);
-            if (logParts.Length != 4)
+            var initialParts = line.Split(new[] { '|' }, 2);
+
```

Comment:
Is it worth checking that we have 2 parts at this stage?

Reply:
yeah, I added that in [775fa69](https://github.com/OctopusDeploy/OctopusTentacle/pull/1048/commits/775fa6917585a8e4f8371c62a0a7b2c2d7d66e86)